ironclust-compiled/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ironclust-base
+
+ENV PATH="/usr/bin/mlrtapp:${PATH}"
+
+RUN echo "version = '5.9.8'\ndate = '03/03/2020'" > /opt/version.txt
+
+RUN apt-get update -y
+RUN apt-get install software-properties-common -y
+RUN add-apt-repository ppa:deadsnakes/ppa -y
+RUN apt-get install git python3.8 python3.8-dev -y
+RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.6 1
+RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 2
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 10
+RUN apt-get install python3-pip -y
+RUN update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1
+
+RUN pip install -U pip
+
+# Installing SI dependencies
+RUN pip install cython numpy \
+    h5py pandas scipy scikit-learn matplotlib networkx \
+    neo joblib tqdm probeinterface
```

Comment:
maybe we use a requirements.txt here?

Reply:
@luiztauffer @chyumin the idea is to keep the sorter-specific as light as possible. These packages wil be installed on the fly with spikeinterface, so I suggest to remove them

Reply:
@alejoe91 but we need SI installed in the image

Reply:
also, why they need to be installed on the fly? Can't we have the image all prepared already?

Reply:
@alejoe91 I was getting some import errors without installing these packages on build, I'll try to reduce these packages as much as I can

Reply:
No, because the idea is that the correct version of SI (and dependencies) that you are running locally is also installed in the docker image. We could also prepare some images with SI installed separately, but this base image should only have the sorter installed

Reply:
Got it. Makes sense